roles/test_operator/defaults/main.yml

```diff
@@ -45,6 +45,32 @@ cifmw_test_operator_tempest_network_attachments: []
 cifmw_test_operator_tempest_tests_include_override_scenario: false
 cifmw_test_operator_tempest_tests_exclude_override_scenario: false
 cifmw_test_operator_tempest_workflow: []
+
+# Enabling SRBAC by default, in jobs where this does not make sense should be turned off explicitly
+cifmw_tempest_tempestconf_config_defaults:
+  deployerInput: |
+    [enforce_scope]
+    nova = true
+    neutron = true
+    glance = true
+    cinder = true
+    keystone = true
+    placement = true
+    manila = true
+
+    [barbican_rbac_scope_verification]
+    enforce_scope = true
+
```

Comment:
I think the barbican one has changed.  @dmendiza   should verify this, but I think we now use the same mechanism [enforce_scope] barbican = true  as everyone else.

Reply:
And what about ironic, ironic_inspector, octavia, designate ?  Are the load_balancer options still correct? @johnsom ?

Reply:
That said, also , I'm ok with merging and amending later as needed.